VCFv4.4.tex

```diff
@@ -625,6 +625,20 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=SVCLAIM,Number=1,Type=String,Description="Claim made by the structural variant call. Valid values are CN, BP, CNBP for copy number, breakpoint and both respectively.">
+\end{verbatim}
+\normalsize
+
+This field disambuates the claim being made by variant caller for DEL, INS, DUP structural variants.
+CN indicates the claim is a copy number claim and the call has been made based on a measure of DNA abundance of the called region.
+BP indicates the claim is a breakpoint claim and the call has been made based on the detection of a non-reference DNA adjacency.
+CNBP indicates the claim is both a copy number and breakpoint claim.
```

Comment:
@d-cameron @jmmut I suggest including a short explanation about why this is necessary. It took me several weeks of discussions to understand this very complex topic, so a regular VCF user might be confused otherwise.

I also rewrite the claim types as a list as suggested by @lbergelson.

```suggestion

The distinction between different claim types is made necessary by the fact that many structural variants affect both the overall copy number of a region and its adjacency structure, but often only one of those effects is being directly observed. It is important to specify which one, because if this is not done, other events in a genome may make the call ambiguous.

For example, a “conventional” deletion usually both decreases the total copy number of a region and creates an adjacency between two sequence locations, which are the breakpoints of that deletion. However, if a deletion is accompanied by a compensatory event elsewhere in the genome, it may only cause changes in the adjacency structure, but not the overall copy number. On the contrary, combinations of certain complex events in a genome may cause the overall copy number decrease, but not the linear excision of a sequence fragment.

To address this, three claim types are defined:

\begin{itemize}
  \item CN (copy number) claim indicates that the call has been made based only on a measure of DNA abundance of the called region, with no evidence to support changes in breakpoint structure.
  \item BP (breakpoint) claim indicates that the call has been made based on the detection of a non-reference DNA adjacency, with no evidence to support overall changes in DNA abundance.
  \item CNBP indicates that there is evidence for both DNA abundancy and adjacency changes, which are consistent with each other and suggest the structural variant of the type being reported.
\end{itemize}
```

Reply:
+1000 except that the style used for quotes in the spec is different:
```
a ``conventional'' deletion
```

 https://github.com/samtools/hts-specs/blob/master/VCFv4.3.tex#L37

Reply:
@jmmut Thanks a lot! I updated the suggestion.

My last hands on experience with LaTeX was in 2013, so I was certain I would mix up some formatting things :)